### PR TITLE
[10.0][IMP] shopinvader_delivery_carrier: Adds carrier descriptions

### PR DESCRIPTION
**Pull request by rousseldenis**
Adds description, description_sale and description_picking in
delivery service in order to enhance customer UX.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@Cedric-Pigeon

File: shopinvader_delivery_carrier/tests/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-from . import test_carrier
-from . import test_notification
-from . import test_delivery_carrier
+# from . import test_carrier
```
Comment (Cedric-Pigeon) [inline shopinvader_delivery_carrier/tests/__init__.py]: @rousseldenis is it expected?

**Review by Cedric-Pigeon — CHANGES_REQUESTED** [review]

File: shopinvader_delivery_carrier/tests/__init__.py (hunk 1 above)
Comment (rousseldenis) [reply shopinvader_delivery_carrier/tests/__init__.py]: Nope indeed !

**Comment by sebastienbeau** [comment]
Hum, I just not sure that it make seen to export so much field
- description_picking is normally use for "stockable" product in order to have a specific name in the delivery
- description is for internal purpose
- description_sale is use on sale order

So maybe just exporting description_sale and map it to description (in the API) is maybe enough?

I fear that front dev ask us too much question "what the fucking hell you have 3 descriptions? which is the good one?"

What do you think?

**Comment by rousseldenis** [comment]
@Cedric-Pigeon An advice on this ?

**Comment by sebastienbeau** [comment]
@rousseldenis @Cedric-Pigeon still valid?

**Comment by lmignon** [comment]
Closing since 10.0 is no more actively maintained.

**Pull request closed without merging**